worker-message/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod message;
+pub mod worker;
+
+pub use message::*;
```

Comment:
> I'd have probably just inlined message into lib.rs.

Good idea.

Reply:
Note that if we move coordinator logic into this crate, it may make sense to keep the messages in their own module. It's also possible that having both the worker and coordinator in one crate may make your original idea of re-exporting some high-level types make more sense...

```rust
mod messages;
pub mod worker;
pub mod coordinator;

// *maybe* these? depends on how much we can use without bringing in `worker::{blah, foo}` anyway.
pub use worker::Worker;
pub use coordinator::Coordinator;
```